javascript-bank/bank.js

```diff
@@ -0,0 +1,33 @@
+/* exported Bank */
+function Bank() {
+  this.nextAccountNumber = 1;
+  this.accounts = [];
+}
+
+Bank.prototype.openAccount = function (holder, balance) {
+  if (Number.isInteger(balance) && balance > 0) {
+    var newAccount = new Account(this.nextAccountNumber, holder);
+    newAccount.deposit(balance);
+    this.nextAccountNumber++;
+    this.accounts.push(newAccount);
+    return newAccount.number;
+  } else {
+    return null;
+  }
+};
+
+Bank.prototype.getAccount = function (number) {
+  if (this.accounts.length + 1 > number) {
+    return this.accounts[number - 1];
```

Comment:
You should not use the `id` for math like this. Instead, you should loop over your list of accounts, and find the one with a matching id.